src/main/java/org/sagebionetworks/web/client/widget/evaluation/EvaluationListViewImpl.java

```diff
@@ -1,81 +1,50 @@
 package org.sagebionetworks.web.client.widget.evaluation;
 
-import com.google.gwt.safehtml.shared.SafeHtmlUtils;
-import com.google.gwt.user.client.ui.InlineHTML;
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
-import java.util.ArrayList;
 import java.util.List;
-import org.gwtbootstrap3.client.ui.ButtonGroup;
-import org.gwtbootstrap3.client.ui.Panel;
-import org.gwtbootstrap3.client.ui.Radio;
-import org.gwtbootstrap3.client.ui.html.Div;
-import org.gwtbootstrap3.client.ui.html.Span;
-import org.gwtbootstrap3.client.ui.html.Text;
 import org.sagebionetworks.evaluation.model.Evaluation;
 import org.sagebionetworks.web.client.DisplayUtils;
-import org.sagebionetworks.web.client.widget.HelpWidget;
+import org.sagebionetworks.web.client.context.SynapseReactClientFullContextPropsProvider;
+import org.sagebionetworks.web.client.jsinterop.AvailableEvaluationQueueListProps;
+import org.sagebionetworks.web.client.jsinterop.React;
+import org.sagebionetworks.web.client.jsinterop.ReactNode;
+import org.sagebionetworks.web.client.jsinterop.SRC;
+import org.sagebionetworks.web.client.widget.ReactComponentDiv;
 
-public class EvaluationListViewImpl
-  extends Panel
-  implements EvaluationListView {
+public class EvaluationListViewImpl implements EvaluationListView {
 
-  List<Radio> evaluationCheckboxes;
+  SynapseReactClientFullContextPropsProvider propsProvider;
+  ReactComponentDiv reactContainer;
+  Presenter presenter;
 
   @Inject
-  public EvaluationListViewImpl() {
-    evaluationCheckboxes = new ArrayList<Radio>();
+  public EvaluationListViewImpl(
```

Comment:
Display logic is handled by the new SRC component, so just render that component here